src/main/resources/templates/toolsPf2e/deity2md.txt

```diff
@@ -9,8 +9,8 @@ tags:
 {/if}
 aliases: ["{resource.name}"{#each resource.aliases}, "{it}"{/each}]
 ---
-# {resource.name}{#if resource.aliases } ({#each resource.aliases}{it}{#if it_hasNext}, {/if}{/each}){/if} *({resource.alignment})*  
-*Source: {resource.source}*  
+# {resource.name}{#if resource.aliases } ({#each resource.aliases}{it}{#if it_hasNext}, {/if}{/each}){/if} *({resource.alignment})*
+*Source: {resource.source}*
```

Comment:
Markdown is sometimes in strict mode.. the two trailing spaces here were to ensure a line break for when strict markdown is used.

Reply:
Done, thank you!